modules/rtdModule/index.js

```diff
@@ -128,6 +128,31 @@ function getProviderData(adUnits, callback) {
   }
 }
 
+/**
+ * delete invalid data received from provider
+ * this is to ensure working flow for GPT
+ * @param {Object} data received from provider
+ * @return {Object} valid data for GPT targeting
+ */
+function validateProviderDataForGPT(data) {
```

Comment:
No tests are added for this function.  It appears the rtd module itself has no tests and only the sub-modules are tested which shouldn't be the case.

Reply:
I added a test for this function

The RTD module and sub-modules are tested together in the same test for now

RTD module has over 80%  test coverage